package.json

```
@@ -47,6 +47,7 @@
   },
   "scripts": {
     "test": "tap",
+    "test-only": "tap",
```

```suggestion
    "test": "npm run test-only",
    "test-only": "tap",
```
i usually do this, to ensure that there's only one source of truth for the test command

---

Good call.  You should make it `npm run test-only --` with the `--` in there though, so that `npm test -- --some-tap-arg` works as expected.

---

https://github.com/npm/arborist/pull/244/commits/b6d4e55dac81afd306a58db2984b6a7342c43247